libtrip-notification/src/main/java/com/mapbox/navigation/trip/notification/MapboxTripNotification.kt

```diff
@@ -306,9 +307,12 @@ class MapboxTripNotification constructor(
                             ?: ManeuverModifier.RIGHT
                     )
                 }
-            }
-            setFreeDriveMode(false)
-        } ?: setFreeDriveMode(true)
+            } ?: updateManeuverImage(
+                    routeProgress.currentLegProgress()?.currentStepProgress()?.step()?.drivingSide()
+                            ?: ManeuverModifier.RIGHT
```

Comment:
Not actual anymore, but anyway, answering for your question: we need to pass any ManeuverModifier to updateManeuverImage method... and I think RIGHT is selected because it is the most popular driving direction for the most countries